### DISCUSSION TITLE
Hillshading Mapsforge

### DISCUSSION THREAD

**Discussion by troNpo** [mapsforge/vtm#1089] [Q&A] [answered]
Hello, in the rendertheme for example "Elevate" if I add the attribute "magnitude" to the line <hillshading... > can I modify  the value of the mapsforge shading. 
Is it possible to do something similar for vtm rendertheme ?
Thanks a lot

**Comment by devemux86** [comment, marked as answer]
Hillshading is not yet implemented in VTM, so this theme rule is not operational.

If anyone can contribute hill shading to VTM, I'm happy to review pull requests. 🙂

**Reply by troNpo** [reply]
It's a pity, in Oruxmaps I can activate the shading of the mapsforge libraries and it manifests itself in the vtm themes, but it's too soft.
Thank you 
![OM_SCREENSHOT](https://github.com/mapsforge/vtm/assets/97335264/9144de71-efc3-45a7-8548-0cbc862fbd95)

**Reply by devemux86** [reply]
Does Orux also use the VTM map engine or only the Mapsforge map engine and themes?

**Reply by troNpo** [reply]
Hi, I'm not sure, Oruxmaps GP uses what in the settings is called a vtm viewer, 
![IMG_20231229_081000](https://github.com/mapsforge/vtm/assets/97335264/4938e399-5dcd-4fc9-ac45-ec0e87d0e9fb)
It can be used on any map. 
It also has the updated vtm libraries and vtm    
 themes that are in your repositories. 
We can use mapsforge maps, mapsforge themes, and vtm themes, with the virtues of up-to-date mapsforge libraries. 
In Oruxmaps we can use our own vtm themes modified from the temes in the repository. 
There is all my interest currently :) 

Thank you
